app/lib/import_export_constants.rb

```diff
@@ -373,7 +373,8 @@ module ImportExportConstants # rubocop:todo Metrics/ModuleLength
     history_type: 'History Type',
     comment: 'History Comment',
     created_at: 'History Created Date',
-    updated_at: 'History Updated Date'
+    updated_at: 'History Updated Date',
+    was_edited: 'Has Been Edited'
```

Comment:
is this language ('Has Been Edited') confirmed with the PO team?

Reply:
@ashankland?

Reply:
It has not: the was_edited field got added after this item was ran by the PO team I believe.

Reply:
also obsolete